CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## History
 
 - [Previous Changelogs](https://github.com/eclipse-theia/theia/tree/master/doc/changelogs/)
+## 1.62.0 - 
+
+<a name="breaking_changes_1.62.0">[Breaking Changes:](#breaking_changes_1.62.0)</a>
+
+- [monaco] Implement "code actions on save" [#15555](https://github.com/eclipse-theia/theia/pull/15555)
```

Comment:
Would likely be good to give some idea of the breakage, mainly onWillSave -> registerSaveListener